.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea/*
 !.idea/icon.svg
+!.idea/dictionaries
```

Comment:
please no, make the exact file, or else we will collect a lot of files over time

Reply:
done x2